src/tools/ceph_monstore_tool.cc

```diff
@@ -877,7 +875,7 @@ int main(int argc, char **argv) {
       goto done;
     }
 
-    if (readable) {
+    if (op_vm.count("help")) {
```

Comment:
s/help/readable/ ?

Reply:
sorry. I make a mistake. I have repushed it.